source/includes/_accounts.md

```diff
@@ -39,18 +41,27 @@ This endpoint creates an account
 
 ### Parameters
 
+Parameter | Required | Default | Description
+--------- | -------  | ------- | ----------
+number | true | N/A | The account number
```

Comment:
Thoughts on formatting this like a table? I think there's a vim plugin that helps